src/components/element-card/element-card.tsx

```diff
@@ -25,9 +34,35 @@ export const ElementCard = ({ element, onClick }: IProps) => {
     const { label, id } = element;
     const { focusedElement, deleteElement } = useBuilder();
 
+    const { isDragging } = useDraggable({
+        id: element.internalId,
+    });
+    const { attributes, listeners, setNodeRef, transform, transition } =
+        useSortable({ id: element.internalId });
+    const { isOver, setNodeRef: droppableRef } = useDroppable({
+        id: element.internalId,
+    });
+
     const isFocused = checkIsFocused();
     const disableDuplicate = shouldDisableDuplicate();
 
+    const style: CSSProperties = {
+        transform: CSS.Translate.toString(transform),
+        transition,
+        opacity: isDragging ? "70%" : "100%",
+        background: isDragging ? "white" : "inherit",
+        gap: isDragging ? "1rem" : "inherit",
+        width: "100%",
+        position: isDragging ? "absolute" : "relative",
```

Comment:
absolute positioning will cause your dragged card to not take up any space too. this causes your droppable content to go under other cards:

<img width="667" alt="image" src="https://github.com/LifeSG/web-form-builder/assets/60728851/c46121aa-44ee-4caf-a36f-6b746e432346">

so using relative positioning is good enough already

```suggestion
        position: "relative",
```